core/workspace.code-workspace

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
This file seems to be an artifact from your local VS-Code editor and should not be part of your MR

Reply:
Yes, it got in by mistake, probably when doing the force push...